management-api-common/src/main/java/com/datastax/mgmtapi/shims/CassandraAPI.java

```diff
@@ -30,6 +30,12 @@
 public interface CassandraAPI
 {
     void decommission(boolean force) throws InterruptedException;
+    
+    void enableFullQuerylog() throws UnsupportedOperationException;
+
+    void disableFullQuerylog()throws UnsupportedOperationException;
```

Comment:
I don't have a strong opinion here, but you could condense these two into a `setFullQueryLog(boolean enabled)` API, like you did with the actual REST endpoint. Either way is fine though.

Reply:
Also, you will have to implement these API methods in the DSEE agent [here](https://github.com/Miles-Garnsey/management-api-for-apache-cassandra/blob/miles/full-query-logging-endpoint/management-api-agent-dse-6.8/src/main/java/com/datastax/mgmtapi/shim/DseAPI68.java#L66)

By default, it won't try to build the DSE agent (as you have to have access to the Datasatx DSE Maven artifactory repo to get the DSE jars). To build the DSE agent, you have to enable the DSE Maven profile, so something like:

```
mvn clean verify -P dse -DrunDSEtests
```

Reply:
DSE part implemented. 

I also thought about where we should split things off into multiple enable/disable functions. I don't think there's a right answer, so I'm happy doing it this way.